### PR TITLE
Changelog: 6.4.3 release

### PR DESCRIPTION
**Pull request by papagian**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by marefr — CHANGES_REQUESTED** [review]
LGTM but would consider adding the following fixes to the changelog:
https://github.com/grafana/grafana/pull/19822
https://github.com/grafana/grafana/pull/19804

**Comment by hugohaggmark** [comment]
Also consider: 
#19699 
#19809

**Review by marefr — APPROVED** [review]
LGTM ContextMenu caould have been under under bug fixes, but probably lacks the type/bug label.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+* **ContextMenu**: make ContextMenu positioning aware of the viewport width. [#19699](https://github.com/grafana/grafana/pull/19699), [@krvajal](https://github.com/krvajal)
+* **DataLinks**: Fix context menu not showing in singlestat-ish visualisations. [#19809](https://github.com/grafana/grafana/pull/19809), [@dprokop](https://github.com/dprokop)
+* **DataLinks**: Fix url field not releasing focus. [#19804](https://github.com/grafana/grafana/pull/19804), [@aocenas](https://github.com/aocenas)
+* **Datasource**: Fixes clicking outside of some query editors required 2 clicks. [#19822](https://github.com/grafana/grafana/pull/19822), [@aocenas](https://github.com/aocenas)
+* **Panels**: Fixes default tab for visualizations without Queries Tab. [#19803](https://github.com/grafana/grafana/pull/19803), [@hugohaggmark](https://github.com/hugohaggmark)
+* **Singlestat**: Fixed issue with mapping null to text. [#19689](https://github.com/grafana/grafana/pull/19689), [@torkelo](https://github.com/torkelo)
+* **grafana/toolkit**: Don't fail plugin creation when git user.name config is not set. [#19821](https://github.com/grafana/grafana/pull/19821), [@dprokop](https://github.com/dprokop)
```
Comment (marefr) [inline CHANGELOG.md]:
```suggestion
* **@grafana/toolkit**: Don't fail plugin creation when git user.name config is not set. [#19821](https://github.com/grafana/grafana/pull/19821), [@dprokop](https://github.com/dprokop)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+* **grafana/toolkit**: TSLint line number off by 1. [#19782](https://github.com/grafana/grafana/pull/19782), [@fredwangwang](https://github.com/fredwangwang)
```
Comment (marefr) [inline CHANGELOG.md]:
```suggestion
* **@grafana/toolkit**: TSLint line number off by 1. [#19782](https://github.com/grafana/grafana/pull/19782), [@fredwangwang](https://github.com/fredwangwang)
```

**Review by marefr — CHANGES_REQUESTED** [review]
While I'm at it. Added two suggestions of changes

**Review by marefr — APPROVED** [review]
LGTM

**Pull request merged**